pkg/scheduler/api/queue_info.go

```diff
@@ -24,11 +24,29 @@ package api
 
 import (
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog/v2"
 
 	"volcano.sh/apis/pkg/apis/scheduling"
 	"volcano.sh/apis/pkg/apis/scheduling/v1beta1"
 )
 
+// Dequeue strategy constants
+const (
+	// DequeueStrategyFIFO means if the first job in queue cannot be dequeued,
+	// the system will repeatedly try to dequeue the first job without skipping
+	DequeueStrategyFIFO = "fifo"
```

Comment:
I think could we set `dequeueStrategy` as a standard field of the queue and put these consts into apis repo? I think it worth it since many users have reported need it. Also /cc @hwdef @hzxuzhonghu @kevin-wangzefeng @wangyang0616 @lowang-bh

Reply:
Agree.

Reply:
@JesseStutler @hwdef  I have set dequeueStrategy as a standard field of the queue https://github.com/volcano-sh/apis/pull/200 ,  Please review this thanks.